wcfsetup/install/files/lib/page/MediaPage.class.php

```diff
@@ -92,9 +92,9 @@ public function readData() {
 			'filesize' => $filesize,
 			'showInline' => in_array($mimeType, self::$inlineMimeTypes),
 			'enableRangeSupport' => $this->thumbnail ? true : false,
-			'lastModificationTime' => $this->media->uploadTime,
+			'lastModificationTime' => $this->media->fileUpdateTime ?? $this->media->uploadTime,
 			'expirationDate' => TIME_NOW + 31536000,
-			'maxAge' => 31536000
+			'maxAge' => 31536000,
```

Comment:
This long of a max age is dangerous if files are able to be replaced.

Likewise the ETag must change after replacement as well.